src/ocamlorg_frontend/pages/blog.eml

```diff
@@ -89,5 +90,8 @@ Layout.render
                 <div><a href="<%s Url.news %>" class="link-orange">See all</a></div>
             </div>
         </div>
+        <div class="prose mt-8">
+            <p style="display: inline;">If you want to contribute a blog post, check out this <a href="https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#blog" target="_blank">Contribute Guide</a> on GitHub.</p>
+        </div>
```

Comment:
```suggestion
        <div class="prose mt-8">
            <p style="display: inline;">If you want to contribute a blog post, or add your RSS feed, check out the <a href="https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#blog" target="_blank">Contributing Guide</a> on GitHub.</p>
        </div>
```

Reply:
maybe try and be consistent "check out the Contributing Guide".

Reply:
@sabine  Thank you. I made the change to 'check out this contributing Guide' for consistency, as I noticed a similar style used in the example on the jobs page. However, if you prefer to check out the contributing Guide' for all links I am more than happy to make that change as well.